src/index.js

```diff
@@ -41,12 +41,18 @@ ReactGA.initialize('UA-532159-1');
 
 const history = createBrowserHistory();
 
-// Get the current location
-history.listen((location) => {
+const trackNavigation = (location) => {
   const locationPage = location.hash && location.hash[0] === '#' && location.hash.slice(1);
   ReactGA.set({ page: locationPage });
   ReactGA.pageview(locationPage);
   window._dha && window._dha.track({ navigatedPage: locationPage });
```

Comment:
I understand that removing send right parameters to DataHub (we should split page, querystring, hash, etc) is out of the scope of this PR.